server/channels/web/saml.go

```diff
@@ -185,6 +196,36 @@ func completeSaml(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	c.App.AttachSessionCookies(c.AppContext, w, r)
 
+	desktopToken := relayProps["desktop_token"]
+	if desktopToken != "" {
+		expiryTime := time.Now().Add(-model.DesktopTokenTTL).Unix()
+		desktopTokenErr := c.App.AuthenticateClientDesktopToken(desktopToken, expiryTime, user)
```

Comment:
Same comment as above in the oauth, this is not needed because we don't need to store the desktop token at all